### PR TITLE
Checkstyle: add CommentsIndentation rule #319

### PR DESCRIPTION
**Pull request by eugenepeh**
#319
Ready for review

### CODE REVIEW

**Comment by CanIHasReview** [comment]
# v1

@eugenepeh submitted v1 for review.

* [v1 1/2] [Checkstyle: add CommentsIndentation rule](https://github.com/se-edu/addressbook-level4/pull/361/commits/65fad643615c827b0f616817196a9c153abed23b)
* [v1 2/2] [Checkstyle: fix CommentsIndentation rule violations ](https://github.com/se-edu/addressbook-level4/pull/361/commits/240f45c7621d1cab52facf8c41a4df270d55373b)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/27cb4621dca1bf474b2838352a90873f3fbcb4ef...240f45c7621d1cab52facf8c41a4df270d55373b))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/361/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by CanIHasReview** [comment]
# v2

@eugenepeh submitted v2 for review.

* [v2 1/1] [Checkstyle: add CommentsIndentation rule and fix violations](https://github.com/se-edu/addressbook-level4/pull/361/commits/28a89140b08d1e8b8e0ea3e24068bd99badf7bac)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/27cb4621dca1bf474b2838352a90873f3fbcb4ef...28a89140b08d1e8b8e0ea3e24068bd99badf7bac)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level4/compare/fce965413aafd619658787fc12073ab5a2480c14~1...fce965413aafd619658787fc12073ab5a2480c14))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/361/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: src/main/java/seedu/address/ui/BrowserPanel.java [hunk 1]
```diff
@@ -27,8 +27,8 @@
      */
     public BrowserPanel(AnchorPane placeholder) {
         super(FXML);
-        placeholder.setOnKeyPressed(Event::consume); // To prevent triggering events for typing inside the
-                                                     // loaded Web page.
+        // To prevent triggering events for typing inside the loaded Web page.
+        placeholder.setOnKeyPressed(Event::consume);
         FxViewUtil.applyAnchorBoundaryParameters(browser, 0.0, 0.0, 0.0, 0.0);
```
Comment (yamgent) [inline src/main/java/seedu/address/ui/BrowserPanel.java]: Might want to leave a space because the comment only applies to the first line of code, not to all three lines.

**Comment by CanIHasReview** [comment]
# v3

@eugenepeh submitted v3 for review.

* [v3 1/1] [Checkstyle: add CommentsIndentation rule and fix violations](https://github.com/se-edu/addressbook-level4/pull/361/commits/b126984fa4b62bc56dd8e65970613b26a365afc3)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/27cb4621dca1bf474b2838352a90873f3fbcb4ef...b126984fa4b62bc56dd8e65970613b26a365afc3)) ([:chart_with_upwards_trend: Interdiff between v2 and v3](https://github.com/se-edu/addressbook-level4/compare/f29faa1fc47877c02ef7053b0520da5aef200f45~1...f29faa1fc47877c02ef7053b0520da5aef200f45))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/361/3/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: config/checkstyle/checkstyle.xml [hunk 2]
```diff
@@ -302,6 +307,5 @@
       <!-- Checks that there is no whitespace before close parenthesis or after open parenthesis. -->
       <property name="severity" value="warning"/>
     </module>
-
```
Comment (PierceAndy) [inline config/checkstyle/checkstyle.xml]: Change is unrelated to commit?

**Comment by CanIHasReview** [comment]
# v4

@eugenepeh submitted v4 for review.

* [v4 1/1] [Checkstyle: add CommentsIndentation rule](https://github.com/se-edu/addressbook-level4/pull/361/commits/113ab3e4d8f42aaefdb6679b755c5d820cda327b)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/27cb4621dca1bf474b2838352a90873f3fbcb4ef...113ab3e4d8f42aaefdb6679b755c5d820cda327b)) ([:chart_with_upwards_trend: Interdiff between v3 and v4](https://github.com/se-edu/addressbook-level4/compare/e042be3d1c8f0d39c908e7dcda2bfa6731feda93~1...e042be3d1c8f0d39c908e7dcda2bfa6731feda93))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/361/4/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: config/checkstyle/checkstyle.xml [hunk 3]
```diff
@@ -242,6 +242,11 @@
     WHITESPACE CHECKS
     -->
 
+    <module name="CommentsIndentation">
+      <!-- Checks that comments are indented relative to their position in the code -->
+      <property name="severity" value="error"/>
```
Comment (PierceAndy) [inline config/checkstyle/checkstyle.xml]: As mentioned elsewhere, this line can be omitted, and the module can be in a single line.

**Comment by CanIHasReview** [comment]
# v5

@eugenepeh submitted v5 for review.

* [v5 1/1] [Checkstyle: add CommentsIndentation rule](https://github.com/se-edu/addressbook-level4/pull/361/commits/5acf8562704544a1463124a6874583cd5f856e6b)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/a61db1bf27f62cc549eb79be0a4598f2f1994b9b...5acf8562704544a1463124a6874583cd5f856e6b)) ([:chart_with_upwards_trend: Interdiff between v4 and v5](https://github.com/se-edu/addressbook-level4/compare/6a915f86599e15f1fcd4120ff32d0a026f46dae3~1...6a915f86599e15f1fcd4120ff32d0a026f46dae3))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/361/5/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by CanIHasReview** [comment]
# v6

@eugenepeh submitted v6 for review.

* [v6 1/1] [Checkstyle: add CommentsIndentation rule](https://github.com/se-edu/addressbook-level4/pull/361/commits/bada4f4332aab088b13f237b625a893fba567ca5)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/07367fab5054768f3243fb55eba1367d3d67a39b...bada4f4332aab088b13f237b625a893fba567ca5)) ([:chart_with_upwards_trend: Interdiff between v5 and v6](https://github.com/se-edu/addressbook-level4/compare/3218d2990657a0b57b543bb0db2af68f8cf1e0b3~1...3218d2990657a0b57b543bb0db2af68f8cf1e0b3))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/361/6/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>
